usb/esp_tinyusb/Kconfig

```diff
@@ -30,6 +30,7 @@ menu "TinyUSB Stack"
     endmenu
 
     menu "Descriptor configuration"
+        comment "You can provide your custom descriptors in tinyusb_driver_install() call"
```

Comment:
Nitpick: `comment "You can provide your custom descriptors via tinyusb_driver_install()"`